recipes/freesasa/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "freesasa" %}
+{% set version = "2.2.0.post3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/freesasa-{{ version }}.tar.gz
+  sha256: db7e5c8a1deae051198d5c7e1a54f2d001b22a945b524d81cb26cacb7d6bb54f
+
+build:
+  script: USE_CYTHON=1 {{ PYTHON }} -m pip install . -vv
```

This will resolve at least the initial Windows failure:
```suggestion
  script_env:
    - USE_CYTHON=1
  script: {{ PYTHON }} -m pip install . -vv
```